sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/util/BlobExtractionMode.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents.util;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.util.Collection;
+
+/**
+ * Defines which parts of a blob will be indexed by the blob storage indexer.
+ * @see <a href="https://docs.microsoft.com/azure/search/search-howto-indexing-azure-blob-storage" />
+ */
+public class BlobExtractionMode extends ExpandableStringEnum<BlobExtractionMode> {
```

Comment:
I'm missing context here, but is there a reason we want this to be an ExpandableEnum? All I see are a set of strings that are passed to the indexing parameter.